scripts/markdown-link-check-config.json

```diff
@@ -0,0 +1,10 @@
+{
+"httpHeaders": [
+    {
+      "urls": ["https://crates.io"],
+      "headers": {
+        "Accept": "text/html"
```

Comment:
HTTP request on crates.io fails with a `404` error, if the `Accept: text/html` header is not specified.